src/sentry/receivers/outbox/__init__.py

```diff
@@ -32,22 +32,17 @@
     return super().delete(**kwargs)
 ```
 
-### Outbox message delivery
+### Outbox message processing
 
 Outbox messages are delivered periodically (each minute) by the `sentry.tasks.enqueue_outbox_jobs`.
-This task runs in both Control and Region Silos and delivers messages to their destination Silo.
+This task runs in both Control and Region Silos and triggers the `send_signal()` method on outbox
+model records.
 
-Should delivery fail for any reason, it will remain in the outbox until it can be successfully
-delivered. Outbox messages will be delivered by cross-region RPC calls.
+Attached signal handlers are triggered in the 'source' region, and are responsible for doing any
+local changes (recording tombstones) and sending RPC calls to update state on the 'other' region.
```

Comment:
Nit:
Wondering why `source` and `other` are in quotes. Should there be existing definitions for terms like `source`, `destination`, `other`, etc, to make communication clearer?

Reply:
Good point, I can clarify further.